modules/apps/static/required-dependencies/required-dependencies/build.gradle

```diff
@@ -42,8 +42,8 @@ dependencies {
 	compileOnly group: "org.apache.geronimo.specs", name: "geronimo-atinject_1.0_spec", version: "1.1"
 	compileOnly group: "org.apache.geronimo.specs", name: "geronimo-interceptor_1.2_spec", version: "1.1"
 	compileOnly group: "org.apache.geronimo.specs", name: "geronimo-jcdi_2.0_spec", version: "1.3"
-	compileOnly group: "org.apache.geronimo.specs", name: "geronimo-json_1.1_spec", version: "1.2"
-	compileOnly group: "org.apache.johnzon", name: "johnzon-core", version: "1.2.3"
+	compileOnly group: "org.apache.geronimo.specs", name: "geronimo-json_1.1_spec", version: "1.5"
```

Comment:
@martamedio How is this related to your commit message "Standarize geronimo-json version numbers" ?